test/react-native-cli/features/cli-tests/automate-symbolication.feature

```diff
@@ -25,11 +25,11 @@ Scenario: successfully modify project
         """
     And I wait for the current stdout line to match the regex "Hit enter to continue"
     When I input a return interactively
-    And I wait for the current stdout line to match the regex "Do you want to automatically upload JavaScript source maps as part of the Gradle build?"
+    And I wait for the current stdout line to match the regex "Do you want to automatically upload JavaScript source maps as part of the Gradle build\?"
     When I input a return interactively
-    And I wait for the current stdout line to match the regex "If you want the latest version of @bugsnag/source-maps hit enter, otherwise type the version you want"
+    And I wait for the current stdout line to match the regex "If you want the latest version of @bugsnag\/source-maps hit enter, otherwise type the version you want"
     When I input a return interactively
-    Then I wait for the shell to output a match for the regex "@bugsnag/source-maps dependency is installed" to stdout
+    Then I wait for the shell to output a match for the regex "@bugsnag\/source-maps dependency is installed" to stdout
```

Comment:
Sorry to be annoying, but these forward slashes don't actually need to be escaped because of how the regex is constructed, e.g.

```ruby
irb(main):001:0> a = Regexp.new("http:\/\/example\.com")
irb(main):002:0> a.match? "http://example.com"
=> true
irb(main):003:0> b = Regexp.new("http://example\.com")
irb(main):004:0> b.match? "http://example.com"
=> true
```

The URLs especially get a bit hard to read with the escaping